docs/superpowers/specs/2026-03-21-channel-api-abstraction-issue-draft.md

```
@@ -0,0 +1,179 @@
+
+### 1. Inconsistent Structure
+
+| Channel | Current Structure | Issue |
+|---------|-------------------|-------|
+| Feishu | `feishu/` (top-level) + `channel/feishu/` (adapter) | Two directories, unclear responsibilities |
+| Telegram | `channel/telegram.rs` | Single file |
+| Matrix | `channel/matrix.rs` | Single file |
+
+### 2. Chaotic Dependencies
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)